tests/unit/_utils/test_requests.py

```diff
@@ -44,18 +44,18 @@ def test_unique_key_to_request_id_matches_known_values(unique_key: str, expected
 @pytest.mark.parametrize(
     ('url', 'expected_output', 'keep_url_fragment'),
     [
-        ('https://example.com/?utm_source=test&utm_medium=test&key=value', 'https://example.com?key=value', False),
+        ('https://example.com/?utm_source=test&utm_medium=test&key=value', 'https://example.com/?key=value', False),
```

Comment:
Do we have to add the slashes here?

Reply:
Yep, `yarl` does not allow `path` to be empty when parameters or fragment is present.